openstudiocore/src/utilities/data/Test/CalibrationResult_GTest.cpp

```diff
@@ -30,8 +30,8 @@ TEST_F(DataFixture, CalibrationResult)
 {
 
   openstudio::path xmlPath = toPath("./calibration_report.xml");
-  if(boost::filesystem::exists(xmlPath)){
```

Comment:
This is like OpenStudio archaeology. I bet we can trim some of this stuff out of utilities for OS 2.0.